libs/rand/random_test.go

```diff
@@ -74,21 +71,6 @@ func testThemAll() string {
 	return out.String()
 }
 
-func TestRngConcurrencySafety(t *testing.T) {
```

Comment:
why remove?

Reply:
I have removed the mutex so this test did not make sense.